### ISSUE TITLE
Additional empty lines in environments

### ISSUE THREAD

**Issue by cdlm** [pillar-markup/pillar#77]
The LaTeX canvas adds too many linebreaks in places, causing LaTeX to break lines when it shouldn't, for instance the environment used for `@@note` blocks:
```
@@note Blah
```
becomes
```
\begin{note}

Blah
\end{note}
```

**Comment by cdlm** [comment]
The canvas probably needs to be stateful to distinguish one newline from several, and the brushes should decide accordingly.

**Comment by DamienCassou** [comment]
looking at it

**Comment by DamienCassou** [comment]
```
Name: Pillar-ExporterLaTeX-DamienCassou.85
Author: DamienCassou
Time: 24 September 2015, 11:20:23.822192 am
UUID: 9bac6417-d5a0-43f8-8159-3efe94d5d6ef
Ancestors: Pillar-ExporterLaTeX-DamienPollet.84

Fix #77: avoid trailing newline when exporting annotated paragraphs
```
